editor/src/components/shared/project-components.ts

```diff
@@ -92,23 +92,13 @@ export function insertableComponentGroupHTML(): InsertableComponentGroupHTML {
   }
 }
 
-export interface InsertableComponentGroupConditionals {
```

Comment:
I grouped conditionals and fragments into one `CodeConstructs` bucket, hope that won't cause issues elsewhere?

Reply:
Looks fine to me 👍